lib/utils.sh

```diff
@@ -24,37 +27,40 @@ get_install_path() {
   local plugin=$1
   local install_type=$2
   local version=$3
-  mkdir -p $(asdf_dir)/installs/${plugin}
+  mkdir -p "$(asdf_dir)/installs/${plugin}"
 
-  if [ $install_type = "version" ]
+  if [ "$install_type" = "version" ]
   then
-    echo $(asdf_dir)/installs/${plugin}/${version}
+    echo "$(asdf_dir)"/installs/"${plugin}"/"${version}"
   else
-    echo $(asdf_dir)/installs/${plugin}/${install_type}-${version}
+    echo "$(asdf_dir)"/installs/"${plugin}"/"${install_type}"-"${version}"
```

Comment:
In both of these cases double quotes around the entire path should suffice.

Reply:
Sounds good. There may be more instances of this as well. I'll poke around before committing the code.